src/Sulu/Component/SmartContent/ContentType.php

```diff
@@ -215,8 +216,12 @@ public function getContentData(PropertyInterface $property)
         $filters = $property->getValue();
         $filters['excluded'] = [$property->getStructure()->getUuid()];
 
+        // extends selected filter with requested tags
+        $filters = array_merge(['tags' => []], $filters);
```

Comment:
hmm i cant think that this is faster than the current solution but for me its ok!